src/components/attribution/AttributionDirective.js

```diff
@@ -35,8 +40,60 @@ goog.provide('ga_attribution_directive');
                 '</div></div>'
           });
         }
+        var update = function(element, layers) {
+          var attrs = {}, list = [], is3dActive = scope.is3dActive();
+          if (is3dActive) {
+            layers = [scope.ol3d.getCesiumScene().terrainProvider].
+                concat(layers);
```

Comment:
With this construct two arrays are created, the one you explicitely create and call `concat` on, and the one created and returned by `concat`. You can do the same with only one array:

```js
    var tmp = [scope.ol3d.getCesiumScene().terrainProvider];
    Array.prototype.push.apply(newLayers, layers);
    layers = tmp;
```